Sources/HipMobileUI/Pages/ExhibitRoutePreviewPage.xaml.cs

```diff
@@ -51,8 +51,10 @@ protected override void OnSizeAllocated(double width, double height)
                 OuterGrid.ColumnDefinitions.Clear();
                 OuterGrid.ColumnDefinitions.Add(new ColumnDefinition { Width = new GridLength(1, GridUnitType.Star) });
                 OuterGrid.RowDefinitions.Clear();
-                OuterGrid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(0.5, GridUnitType.Star) });
-                OuterGrid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(0.5, GridUnitType.Star) });
+                //The image got a ratio of 9:7
+                var imageGridHeight = width / height / 9 * 7;
```

Comment:
Since you're using it in several places, maybe you can extract `7.0/9.0` into a constant and multiply `width / height` with it, which would also allow this to be changed consistently across all screens if ever needed.